### PR TITLE
Update notebook

### PR DESCRIPTION
**Pull request by weiglszonja**
Update notebook to utilise methods from `neuroconv` where possible.

The converted file using this notebook is here: https://drive.google.com/file/d/1MhnfKcG_zpXzNbQSyUDyoaxgy9tD4rgI/view?usp=sharing

### CODE REVIEW

**Review by CodyCBakerPhD — DISMISSED** [review]

**Review by CodyCBakerPhD — DISMISSED** [review]

File: convert_microns_to_nwb.ipynb [hunk 1]
```diff
@@ -27,158 +25,162 @@
...
-    "    \n",
+    "\n",
     "    pupil_maj_r = TimeSeries(\n",
     "        name=\"pupil_maj_r\",\n",
-    "        description=\"jajor axis of pupil tracking ellipse\",\n",
+    "        description=\"major axis of pupil tracking ellipse\",\n",
     "        data=H5DataIO(pupil_maj_r, compression=True),\n",
-    "        timestamps=pupil_min_r,\n",
-    "        unit=\"unknown\",\n",
+    "        timestamps=H5DataIO(timestamps, compression=True),\n",
```
Comment (bendichter) [inline convert_microns_to_nwb.ipynb]: Can we change the timestamps back? This is a PyNWB trick to reuse timestamps: https://pynwb.readthedocs.io/en/stable/tutorials/general/file.html#reusing-timestamps

File: convert_microns_to_nwb.ipynb [hunk 2]
```diff
@@ -27,158 +25,162 @@
...
-    "    treadmill_velocity = TimeSeries(\n",
+    "\n",
+    "    time_series_kwargs = dict(\n",
     "        name=\"treadmill_velocity\",\n",
     "        data=H5DataIO(treadmill_velocity, compression=True),\n",
     "        timestamps=H5DataIO(treadmill_timestamps, compression=True),\n",
-    "        description=\"velocity of treadmill\",\n",
-    "        unit=\"unknown\",\n",
+    "        unit=\"n.a.\",\n",
+    "        description=\"velocity of treadmiszonjaweiglll\",\n",
```
Comment (bendichter) [inline convert_microns_to_nwb.ipynb]: I think you accidentally added your username here

File: convert_microns_to_nwb.ipynb [hunk 3]
```diff
@@ -190,303 +192,320 @@
...
     "        region=list(range(data.shape[1])),\n",
     "        description=f\"all rois in field {field_key['field']}\"\n",
     "    )\n",
-    "    \n",
+    "\n",
+    "    roi_response_series_name = \"RioResponseSeries\" if plane_segmentation_index == 0 else f\"RioResponseSeries{field_key['field']}\"\n",
+    "\n",
     "    roi_response_series = RoiResponseSeries(\n",
-    "        name=f\"RioResponseSeries{field_key['field']}\",\n",
+    "        name=roi_response_series_name,\n",
```
Comment (bendichter) [inline convert_microns_to_nwb.ipynb:394]: I'd rather keep the 0

File: convert_microns_to_nwb.ipynb (hunk 2 above)
Comment (weiglszonja) [reply convert_microns_to_nwb.ipynb]: gosh, thanks for catching that!

File: convert_microns_to_nwb.ipynb (hunk 1 above)
Comment (CodyCBakerPhD) [reply convert_microns_to_nwb.ipynb]: She's going to modularize these helpers as tools in importable `.py` functions to make designing the full conversion script over sessions easier; this change will be much easier to make once they are out of the notebook

File: convert_microns_to_nwb.ipynb (hunk 3 above)
Comment (CodyCBakerPhD) [reply convert_microns_to_nwb.ipynb:394]: She's going to modularize these helpers as tools in importable `.py` functions to make designing the full conversion script over sessions easier; this change will be much easier to make once they are out of the notebook

Comment (bendichter) [reply convert_microns_to_nwb.ipynb:394]: she can't just change it back?

**Comment by CodyCBakerPhD** [comment]
@weiglszonja It might be easier to modularize all this before making core changes to the code, actually - can you do that first?

**Comment by CodyCBakerPhD** [comment]
@weiglszonja Might be easier to just update the notebook with the helper functions in a separate PR now, what do you think? Or do you want to keep this one open?

**Comment by weiglszonja** [comment]
No, we can close this.

**Pull request closed without merging**